packages/runtime-common/module-syntax.ts

```diff
@@ -27,12 +28,11 @@ import type { types as t } from '@babel/core';
 import type { NodePath } from '@babel/traverse';
 import type { FieldType } from 'https://cardstack.com/base/card-api';
 
-export type { PossibleCardClass, Export };
-import { unionWith } from 'lodash';
+export type { PossibleCardClass, BaseDeclaration };
 
 export class ModuleSyntax {
   declare possibleCards: PossibleCardClass[];
-  declare exports: Export[];
+  declare moduleDeclarations: ModuleDeclaration[];
```

Comment:
This is also probably such a bad name. Shall I call it elementDeclaration? the frontend uses the words elements to refer to this

Reply:
I just used ElementDeclaration for the type. And elements for the instance name. I didn't wanna pollute more use of names. Since I've been using elements. I'll just use elements